pkg/option/config.go

```diff
@@ -2056,6 +2076,15 @@ func (c *DaemonConfig) Validate() error {
 		return fmt.Errorf("IPv6 cannot be enabled in ENI IPAM mode")
 	}
 
+	if c.EnableIPv6NDP {
+		if !c.EnableIPv6 {
+			return fmt.Errorf("IPv6NDP cannot be enabled when IPv6 is not enabled")
+		}
+		if len(c.IPv6MCastDevice) == 0 {
+			return fmt.Errorf("IPv6NDP cannot be enabled without ipv6 mcast device")
```

Comment:
```suggestion
			return fmt.Errorf("IPv6NDP cannot be enabled without %s", option.IPv6MCastDevice)
```
That makes it easier for users to know which flag to use when they see this error.